AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/dropdown-menu/justice-user-menu-stub.component.ts

```diff
@@ -14,4 +14,7 @@ export class JusticeUserMenuStubComponent {
     clear() {
         console.log('stub method');
     }
+    enabled(enabled: boolean) {
+        console.log(`Justice User Menu component enabled state changed to : ${enabled}`);
```

Comment:
is it possible to use the logger?

Reply:
this is a stub component for testing, not the real one.

Reply:
ok no worries.